corehq/form_processor/models.py

```diff
@@ -984,28 +982,16 @@ def from_form_attachment(self, attachment):
         self.content_type = attachment.content_type
         self.properties = attachment.properties
 
-        if not self.content_type and self.attachment_src:
-            guessed = mimetypes.guess_type(self.attachment_src)
+        if not self.content_type and attachment_src:
+            guessed = mimetypes.guess_type(attachment_src)
             if len(guessed) > 0 and guessed[0] is not None:
                 self.content_type = guessed[0]
 
-        if isinstance(attachment, CaseAttachmentSQL):
-            assert self.identifier == attachment.identifier
```

Comment:
shouldn't this assert remain as `assert self.name == attachment.name`?

Reply:
I don't think this method is ever called with `CaseAttachmentSQL`, so I don't think this assertion is ever executed.